activation/wire/wire_v2_test.go

```diff
@@ -125,3 +127,21 @@ func Test_NoATXv2IDCollisions(t *testing.T) {
 		atxIDs = append(atxIDs, id)
 	}
 }
+
+func Test_ATXv2_SupportUpTo4Niposts(t *testing.T) {
+	f := fuzz.New()
+	atx := &ActivationTxV2{}
+	f.Fuzz(atx)
+	for i := range 4 {
+		t.Run(fmt.Sprintf("supports %d poet", i), func(t *testing.T) {
+			atx.NiPosts = make([]NiPostsV2, i)
+			_, err := codec.Encode(atx)
+			require.NoError(t, err)
+		})
+	}
+	t.Run("doesn't support > 5 niposts", func(t *testing.T) {
+		atx.NiPosts = make([]NiPostsV2, 5)
+		_, err := codec.Encode(atx)
+		require.Error(t, err)
+	})
+}
```

Comment:
I don't think we need this test? It basically tests that `go-scale` works correctly and we have tests for that in the go-scale repository 🙂